### PR TITLE
add @guangrei

### PR DESCRIPTION
**Pull request by guangrei**
* [X] I would also like a **Thank you PHP** badge on my GitHub profile.

### CODE REVIEW

**Comment by petk** [comment]
@guangrei, this was intentionally closed? In case you still want to get on the list, just re-open the pull request.

**Comment by guangrei** [comment]
> @guangrei, this was intentionally closed? In case you still want to get on the list, just re-open the pull request

> @guangrei, this was intentionally closed? In case you still want to get on the list, just re-open the pull request.

yes, I closed the pull request immediately after the review status failed https://github.com/thank-you-php/thank-you-php/actions/runs/6034926296/job/16374267103?pr=434
I have no idea how to fix this.

File: README.md [hunk 1]
```diff
@@ -424,3 +425,4 @@ Inspired by the
 [Thank You Github](https://github.com/thank-you-github/thank-you-github),
 [Thank You Laravel](https://github.com/thank-you-laravel/thank-you-laravel),
 and [Thank You Symfony](https://github.com/thank-you-symfony/thank-you-symfony).
+
```
Comment (petk) [inline README.md:428]: There is additional final newline added here, that's why it was failing.

**Comment by guangrei** [comment]
@petk Thanks for your advice, I have fixed it and created new pull request 🙏

**Pull request closed without merging**